rtbkit/core/router/router_runner.cc

```diff
@@ -59,7 +59,8 @@ RouterRunner() :
     slowModeTolerance(MonitorClient::DefaultTolerance),
     slowModeMoneyLimit(""),
     analyticsOn(false),
-    analyticsConnections(1)
+    analyticsConnections(1),
+    augmentationWindow(0.05)
```

Comment:
I would love to have type-safe duration types. Actually, C++11 already has it (std::chrono::milliseconds(5)). I think that when we'll definitely move to a recent version of gcc, we could use the chrono literal suffixes so that we'll write:

```
augmentationWindow(5ms)
```